### DISCUSSION TITLE
Allow to upload an image from the image menu

### DISCUSSION THREAD

**Discussion by fredmabire** [sibiraj-s/ngx-editor#316] [Ideas]
It should be nice if we could upload an image directly from the image button.   

![image](https://user-images.githubusercontent.com/34098151/105530861-b55ef780-5ce8-11eb-96b8-d499c66bce80.png)  

Maybe it's already possible... 
Otherwise, there is an example of implementation at [prosemirror.net](https://prosemirror.net/examples/upload/)

**Comment by sibiraj-s** [comment]
HI. @fredmabire Thanks for opening the discussion. I previously had the image upload option, but the requirements keep on changing. And I remember some people asking for custom upload option even-though it is there. Hence the Option is removed. With `CustomMenu`, You should be able to insert a custom menu item that uploads the image and once uploaded you can insert it back to the editor by

```ts
this.editor.commands
  .insertImage('<YOUR IMAGE URL>')
  .exec()
```

[a comment by an automated account is not shown]
